model/sys/mem.sail

```diff
@@ -88,14 +88,14 @@ function phys_mem_read forall 'n, 0 < 'n <= max_mem_access . (t : MemoryAccessTy
 function accessFaultFromAccessType(accTy : MemoryAccessType(ext_access_type)) -> ExceptionType =
   match accTy {
     InstructionFetch() => E_Fetch_Access_Fault(),
-    Load(Data)         => E_Load_Access_Fault(),
+    Load(_)            => E_Load_Access_Fault(),
     _                  => E_SAMO_Access_Fault(),
   }
 
 function alignmentFaultFromAccessType(accTy : MemoryAccessType(ext_access_type)) -> ExceptionType =
   match accTy {
     InstructionFetch() => E_Fetch_Addr_Align(),
-    Load(Data)         => E_Load_Addr_Align(),
+    Load(_)            => E_Load_Addr_Align(),
```

Comment:
Btw I think this technically changes the behaviour - in the CHERI version you could have `Load(Tagged)` or something like that. However... this looks more correct to me anyway.